types/chrome/index.d.ts

```diff
@@ -3316,15 +3316,15 @@ declare namespace chrome.extension {
      * function(any response) {...};
      * Parameter response: The JSON response object sent by the handler of the request. If an error occurs while connecting to the extension, the callback will be called with no arguments and runtime.lastError will be set to the error message.
      */
-    export function sendRequest(extensionId: string, request: any, responseCallback?: (response: any) => void): void;
+    export function sendRequest<T = any, R = any>(extensionId: string, request: T, responseCallback?: (response: R) => void): void;
```

Comment:
could you make the variables more readable?

Reply:
I changed them to `Request` and `Response`. If you have better names in mind, I'd gladly use that.

Reply:
they look good to me! go ahead and mark these conversations as resolved